HoneyBadgers.WebApp/Controllers/ReportController.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+using HoneyBadgers.Entity.Models;
+using Newtonsoft.Json;
+
+namespace HoneyBadgers.WebApp.Controllers
+{
+    public class ReportController : Controller
+    {
+        private IHttpClientFactory _iHttpClientFactory;
+        public ReportController(IHttpClientFactory iHttpClientFactory)
+        {
+            _iHttpClientFactory = iHttpClientFactory;
+        }
+        [HttpGet]
+        public async Task<IActionResult> Index()
+        {
+            var client = _iHttpClientFactory.CreateClient();
+            var request = new HttpRequestMessage(HttpMethod.Get, "https://localhost:5001/api/reports");
+            var result = await client.SendAsync(request);
```

Comment:
Może warto to zabezpieczyć przed wywaleniem aplikacji? W przypadku nieuruchomienia lub nawet przypadkowego zamknięcia projektu REST API aplikacja się nie wysypie. Zwrócić w widoku jakiś komunikat "Brak komunikacji z REST API" lub coś w tym stylu